test-data/apply-configuration/json-patch/expected-output/Management/PatchStatus/cluster-scoped-resources/operator.openshift.io/authentications/001-body-cluster.yaml

```diff
@@ -0,0 +1 @@
+[{"op":"test","path":"/status/conditions/2/type","value":"WebhookAuthenticatorCertApprover_OpenShiftAuthenticatorDegraded"},{"op":"remove","path":"/status/conditions/2"}]
```

Comment:
want to format these in the future, one per line?

Reply:
yes, a good idea, we should do that in the future.